schemas/src/main/java/com/scottlogic/deg/schemas/common/ProfileSerialiser.java

```diff
@@ -10,6 +10,7 @@ public String serialise(BaseProfile profile) throws IOException {
         ObjectMapper mapper = new ObjectMapper();
 
         mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        mapper.setSerializationInclusion(JsonInclude.Include.NON_EMPTY);
```

Comment:
Can you describe how the serialiser would now serialise a constraint where `is` hasn't been set to a value (or null)

Reply:
When I say describe, i mean include the JSON it would emit.